src/skeletons/skeleton-text.tsx

```diff
@@ -0,0 +1,54 @@
+// https://www.figma.com/file/w7E0mmB92eio0zHw7h9iS2/%5BREADY%5D-Skeletons-Specs?node-id=986%3A1161
+
+import * as React from 'react';
+import Stack from '../stack';
+import {useIsInverseVariant} from '../theme-variant-context';
+import {useAnimation} from './styles';
+import {createUseStyles} from '../jss';
+
+type TextStylesProps = {
+    isInverse: boolean;
+};
+
+const useTextStyles = createUseStyles(({colors}) => ({
+    line: {
+        borderRadius: 8,
+        height: 8,
+        width: '100%',
+        background: ({isInverse}: TextStylesProps) =>
```

Comment:
should be this style in skeletons styles across to any skeleton instead of repeat colors in each skeleton?

Reply:
I'm unifying all the components into a single file, so yes, I'll apply DRY